### PR TITLE
daemon: fix log missing

### PR DESCRIPTION
**Pull request by FeelyChau**
If no one subscribes the `data` event, the callback will not be called. See:
https://github.com/nodejs/node/blob/master/lib/_stream_transform.js#L37

### CODE REVIEW

File: packages/daemon/src/service/log-manager.ts [hunk 1]
```diff
@@ -57,6 +54,17 @@ class LogPassthrough extends Transform {
     callback();
   }
 
+  /**
+   * cover Transform.write, otherwise if no `data` event listener,
+   * the callback `_transform` will not be called, but we need to save the log to file.
+   * @param args args for stream.write
```
Comment (yorkie) [inline packages/daemon/src/service/log-manager.ts:60]: This description is incorrect, rest parameters should not be supported by typedoc.

File: packages/daemon/src/service/log-manager.ts [hunk 2]
```diff
@@ -57,6 +54,17 @@ class LogPassthrough extends Transform {
...
+   */
+  write(...args: any[]): boolean {
```
Comment (yorkie) [inline packages/daemon/src/service/log-manager.ts:62]: Shall we have a unit test for this change?

**Comment by FeelyChau** [comment]
Test added.

**Comment by FeelyChau** [comment]
Crushed when test.
```sh
2020-09-02 13:28:50,015 ERROR 45547 nodejs.unhandledRejectionError: EBADF: bad file descriptor, write
unhandledRejectionError: EBADF: bad file descriptor, write
errno: -9
code: "EBADF"
syscall: "write"
name: "unhandledRejectionError"
pid: 45547
hostname: MacBook-Pro.local
```

**Review by yorkie — APPROVED** [review]

**Review by rickycao-qy — APPROVED** [review]
LGTM

**Pull request merged**